docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -85,30 +85,33 @@ With the first release of this new Workflow, existing Projects will require a mi
 
 ## Activating the feature
 
-:::info
-In order to activate the feature on a Project or Company, please open a service request and ask for the support of a Mia-Platform referent.
-:::
+The Enhanced Project Workflow can be activated by the user when [creating a new Project](/console/project-configuration/create-a-project.mdx#creation-process). There are different possible scenarios in the Project creation process:
+- The Company requires that all new Projects are created with the Enhanced Workflow. This means that during the Project creation process the Enhanced Workflow will be the only available workflow option.
+- The Company lets users choose if they want to adopt the Enhanced Workflow on the new Project or keep using the standard one.
```

Comment:
this option is not valid anymore @MIA-Deltat1995 I think this relates to your sprint activity

the same applies to the line below, please rework directly in this PR